resource/labeling-app/components/category-selector.js

```diff
@@ -200,10 +200,19 @@ let CategorySelector = {
                         <i v-if="status === 'SKIPPED'" class="fas fa-forward skipped"></i>
                         <h2 v-if="status === 'SKIPPED'">Sample was skipped</h2>
                     </div>
-                    <div v-else>
-                        <div v-if="status !== 'SKIPPED'" class="circle"></div>
+                    <div v-else style="width: 100%;">
                         <h2 v-if="status !== 'SKIPPED'">No labels yet</h2>
-                        <p>Select a label and draw a box around the target.</p>
+                        <p style="margin: auto;">Select a label and draw a box around the target then...</p>
```

Comment:
Not sure I understand the phrase:
> Select a label and draw a box around the target then... Click on the categories buttons

you actually click on a category button in order to select it. 

![image](https://user-images.githubusercontent.com/7889985/100092484-6ca8d080-2e56-11eb-9d4a-87d071d02a09.png)



I'd reformat it to be:

NO LABELS YET
Select a label by
[bubble 1](Clicking on the categories buttons) or [bubble 2](Using keyboard shortcuts)

... and draw a box around the target

Reply:
also after omitting the "draw a box..." part this same hint can be added to the rest of webapps, not only object labeling

Reply:
I agree the wording here is weird. We could have two different versions between lal and tal going like this:

> Highlight the text to label then apply a category by…  clicking […] Using […]

> Draw a box around the target then apply a category by… clicking […] using […]

Reply:
Changed made in 0fa643f